lib/index.js

```diff
@@ -64,6 +64,31 @@ module.exports = (nextConfig = {}) =>
         ],
       });
 
+      if (config.module.rules) {
```

Comment:
Is this backward-compatible with previous version? (i.e. `@zeit/next-css` )
Otherwise, could you consider adding to a new option?